esp/src/src-react/components/controls/Grid.tsx

```diff
@@ -260,7 +260,7 @@ const FluentStoreGrid: React.FunctionComponent<FluentStoreGridProps> = ({
             setItems(items);
             selectedIndices.forEach(index => selectionHandler.setIndexSelected(index, true, false));
         });
-    }, [count, selectionHandler, start, store], [query, sorted]);
+    }, [abortController?.current?.signal, count, selectionHandler, start, store], [query, sorted]);
```

Comment:
This might be an anti-pattern (having a `Ref` trigger an `effect`)?

Also I wonder if we could convert this to a regular `React.useCallback`?

Reply:
Is there an effect involved? Looking at useDeepCallback, it's a useCallback that has a useDeepCompareMemoize for the second array of dependencies. But I didn't see a useEffect anywhere in there.